server/models/Event.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const eventSchema = new mongoose.Schema({
+  userURL: {
+    type: String,
+    required: true,
+  },
```

Comment:
Would it be better to store the userId here and reference so we can easily grab all the user's events they own. Reason being, what if we want to be able to update their usernames? This would be more work in long-term.

Reply:
UserId will be same as UserURL, as whenever a user is signed up to the application we generate a unique URL to the user! I can simply change it to UserID and also in User model, we need to create an entry for userID/userURL. Do you want me to update UserURL to UserID and create an entry in User model?